src/components/flow/index.js

```diff
@@ -118,6 +119,12 @@ const Flow = forwardRef(
         });
     }, [isAuditLogShown, flowDoc]);
 
+    useEffect(() => {
+      if (nerdletState.flow.isPlayback) {
+        setIsPlayback(nerdletState.flow.isPlayback);
+      }
+    }, [nerdletState.flow.isPlayback]);
```

Comment:
It's safer to not use nested variables in nerdlet state. Use `nerdletState.isPlayback` instead